src/config/dark-sites.config

```diff
@@ -233,6 +235,7 @@ erai-raws.info
 esportal.com
 etcroot.pw
 eternal.abimon.org
+ethermine.org
```

Comment:
Please remove this entry.
![image](https://user-images.githubusercontent.com/25481501/110012494-7aed6f00-7d18-11eb-81f8-321c31789162.png)